tests/State/HubTest.php

```diff
@@ -135,6 +135,30 @@ public function testPopScope(): void
     }
 
     public function testWithScope(): void
+    {
+        $scope = new Scope();
+        $hub = new Hub($this->createMock(ClientInterface::class), $scope);
```

Comment:
Since we don't care about the client we can pass `nulll`

Reply:
Done